Vizzuality/landgriffon
# Impact table pagination

### General description

This PR adds pagination to the impact table. The pagination is applied at API level too.

### Designs

https://www.figma.com/file/Bdxq4bJD6eXFG1BuSjw6gG/Landgriffon---Redesign?node-id=478%3A19064

### Testing instructions

- Go to the impact table
- Select a unit with enough data (region)
- The table should populate with those parameters
- Changing the page size should trigger a new request, the table info should update, including the pagination info
- Changing the page should trigger a refetch for that page as well

## Checklist before merging
